packages/editor/src/components/provider/index.js

```diff
@@ -271,7 +272,7 @@ class EditorProvider extends Component {
 		);
 
 		return (
-			<>
+			<StyledProvider>
```

Comment:
This enables it for the post-editor (probably better moved to edit-post) but not for edit-widgets, edit-site, edit-navigation and list-reusabale-blocks (these are the top level packages).

 - How do we make sure folks don't forget to add these providers? 
 - Is there a way to throw a warning if we detect a missing provider?
 - Should we embed other providers there in a single one? (thinking about popover and slot providers).
 - Should this be marked as a breaking change in the changelog of the components package?